### ISSUE TITLE
upload error: java heap space: zootaxa.4321.1.1

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#257]
again a Java heap space error

here is the IMF file. let me know, when you finish the upload, so I can delete this file

https://drive.google.com/a/plazi.org/file/d/0B_yrQwn4yBySeng0d2Uza3NrOGM/view?usp=sharing

![image](https://user-images.githubusercontent.com/4609956/30579087-64fb5ebc-9d17-11e7-8db1-cdb6592e26c2.png)

![image](https://user-images.githubusercontent.com/4609956/30579093-736049b8-9d17-11e7-8098-225439cbabaa.png)

**Comment by myrmoteras** [comment]
I closed the file, restarted GGI and now it stops somwhere else
![image](https://user-images.githubusercontent.com/4609956/30579585-9aa75848-9d19-11e7-91a5-66e1a363b7d4.png)


then I am saving the file as IMD, restart GGI and get this error

![image](https://user-images.githubusercontent.com/4609956/30579724-145e066e-9d1a-11e7-8444-0fab563d56c0.png)

and again
![image](https://user-images.githubusercontent.com/4609956/30579748-33d013a2-9d1a-11e7-8a1f-ff45d3c31c7e.png)

**Comment by gsautter** [comment]
OK, getting it. What does your task manager give up in terms of memory consumption, in particular for `java.exe` processes?

One possible reason I could think of for these issues is that you might be running Java 32 bit. For technical limitations (address space), 32 bit Java can handle a maximum of 2 GB of memory, with some 1.6 GB available to applications in practice.
On a 64 bit machine, you could easily switch to Java 64 bit, which should solve all the problems right away.

To find out what version of Java you are running, open a dos prompt and enter `java -version`. What do you get?

**Comment by myrmoteras** [comment]
Microsoft Windows [Version 10.0.15063]
(c) 2017 Microsoft Corporation. All rights reserved.

C:\Users\Donat>java -version
java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)

C:\Users\Donat>

**Comment by gsautter** [comment]
OK, that's one ahead of me ... still running Java 1.7 on my machine. However, the differences are so significant I cannot imagine that's the culprit.

**Comment by gsautter** [comment]
Could you check memory consumption in the task manager when those errors occur? I wonder how far it builds up ...

The other thing is what's in your `Parameters.cnfg` ... maybe the memory limit is set too restrictively.

**Comment by myrmoteras** [comment]
you are right. somehow I got a new version of GGI and did not change the config. Now it is set to min 1024, max 10240.
But I still get the same error

![image](https://user-images.githubusercontent.com/4609956/30625016-083ca884-9dc1-11e7-86f5-d416f931fcc3.png)

**Comment by myrmoteras** [comment]
and the other part is that it always stops at figure@59...
![image](https://user-images.githubusercontent.com/4609956/30625079-68f2b9c0-9dc1-11e7-9d7c-8740882cdcc2.png)

**Comment by myrmoteras** [comment]
The good thing though is, that once I closed GGI and restarted it, it worked. So, this file is done.

Conclusion: memory allocation.
